WordPressVIPMinimum/Sniffs/Constants/ConstantStringSniff.php

```diff
@@ -55,7 +56,7 @@ public function process_token( $stackPtr ) {
 			return;
 		}
 
-		$param = $this->get_function_call_parameter( $stackPtr, 1 );
+		$param = PassedParameters::getParameter( $this->phpcsFile, $stackPtr, 1 );
```

Comment:
```suggestion
		$param = PassedParameters::getParameter( $this->phpcsFile, $stackPtr, 1, 'constant_name' );
```

For function calls, the parameter name is (soft) required now to allow for support for PHP 8.0+ function calls passing named parameters.

If the parameter name is not passed and the parameter being requested is not found, the `PassedParameters::getParameter()` method will throw an exception.

In this case, the parameter name for both functions being "sniffed" is the same, so can just be passed like per the suggestion, though if the sniff would at some point need to examine other functions, a different, more flexible, solution would be needed.

Refs:
* https://phpcsutils.com/phpdoc/classes/PHPCSUtils-Utils-PassedParameters.html#method_getParameter
* https://www.php.net/manual/en/function.define.php
* https://www.php.net/manual/en/function.defined.php

Reply:
Updated in 98f52dbd65c31d7574028a8d68419f0232ca9e68.